apps/avifgainmaputil/convert_command.cc

```diff
@@ -0,0 +1,108 @@
+// Copyright 2023 Google LLC
+// SPDX-License-Identifier: BSD-2-Clause
+
+#include "convert_command.h"
+
+#include "avif/avif_cxx.h"
+#include "avifutil.h"
+#include "imageio.h"
+#include "swapbase_command.h"
+
+namespace avif {
+
+ConvertCommand::ConvertCommand()
+    : ProgramCommand("convert", "Convert a jpeg with a gain map to avif.") {
```

Comment:
"convert" is fine unless you plan to add another command that will convert AVIF to something else.

Reply:
If that is ever added, I think the same "convert" command could probably be used